docs/customizations/webchatcontainer.md

```diff
@@ -0,0 +1,501 @@
+| cardActionMiddleware | OneOrMany\<CardActionMiddleware\>  | No | Control the behaviors of certain adaptive cards | [cardActionMiddleware](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/renderingmiddlewares/cardActionMiddleware.ts)
+| children | ReactNode  | No | The children components of WebChat. Not used in this package by default. For more information, please go to WebChat's official documenetation page.| -
+| dir | string  | No | The general locale direction | "ltr"
+| disabled | boolean  | No | Disables the typing and sending area | false
+| downscaleImageToDataURL | (blob: Blob, maxWidth: number, maxHeight: number, type: string, quality: number) => string  | No | Not used in this package by default. For more information, please go to WebChat's official documenetation page. | -
+| grammars | any  | No | Not used in this package by default. For more information, please go to WebChat's official documenetation page. | -
+| groupActivitiesMiddleware | OneOrMany<GroupActivitiesMiddleware> | No | Controls how the activities are grouped together regarding to avatar and timestamp grouping | [groupActivitiesMiddleware](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/renderingmiddlewares/groupActivitiesMiddleware.tsx)
+| internalErrorBoxClass | React.Component \| Function | No | Not used in this package by default. For more information, please go to WebChat's official documenetation page. | -
+| locale | string | No | Sets the locale code for WebChat | "en-US"
+| onTelemetry | any | No | Sets pluggable telemetry logger for WebChat telemetry events | [WebChatLogger](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-widget/src/components/webchatcontainerstateful/webchatcontroller/webchattelemetry/WebChatLogger.ts)
+| overrideLocalizedStrings | any | No | Overrides default strings in webchat. This is not used by default since we modify the strings in the various middlewares themselves | -
+| renderMarkdown | any | No | Sets the callback function for markdown renderer | [renderMarkdown](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-widget/src/components/livechatwidget/common/initWebChatComposer.ts#L99)
+| scrollToEndButtonMiddleware | OneOrMany<ScrollToEndButtonMiddleware> | No | Controls the behaviors of the "scroll to end" button. This is not used by default | -
+| selectVoice | (voices: typeof window.SpeechSynthesisVoice[], activity: DirectLineActivity) => void | No | Not used in this package by default. For more information, please go to WebChat's official documenetation page. | -
+| sendTypingIndicator | boolean | No | Whether to send typing indicator the message recipient | true
```

Comment:
grammar: `Whether to send typing indicator the message recipient` -> `Whether to send typing indicator to the message recipient`?